apps/meteor/server/lib/getMaxLoginTokens.ts

```diff
@@ -0,0 +1,5 @@
+export const maxLoginTokens = parseInt(String(process.env.MAX_RESUME_LOGIN_TOKENS)) || 50;
+
+export function getMaxLoginTokens(): number {
```

Comment:
I can move this to the new `User` service as well, but since it is actually static I didn't want to add any extra steps into the stack. but since it is an internal service, which doesn't make network calls, I guess it is safe to move it to the service.. wdyt?

Reply:
I think it depends on who's gonna use it. If the user service would be the main consumer, then I think it would be better there. If there's gonna be other users outside of that service, then a lib could be better.